Mathlib/MeasureTheory/Constructions/Polish.lean

```diff
@@ -558,7 +558,7 @@ theorem measurableSet_preimage_iff_of_surjective [CountablySeparated Z]
     exact h.compl.analyticSet_image hf
 #align measurable.measurable_set_preimage_iff_of_surjective Measurable.measurableSet_preimage_iff_of_surjective
 
-theorem map_measurableSpace_eq  [CountablySeparated Z]
+theorem map_measurableSpace_eq [CountablySeparated Z]
```

Comment:
I don't object to these changes.

I'm not sure if doing them by hand is time-efficient, given that a Lean formatter doesn't seem to far away...
if you feel like shepherding more of these, I have a [branch](https://github.com/leanprover-community/mathlib4/tree/MR-formatting-nits) doing a few hundred of lines of such mathlib replacements - let me know if you're interested in reviewing such a PR. Otherwise, I'm happy to wait for a formatter instead.

Reply:
I'm not holding my breath for a formatter :) I personally wouldn't mind landing automatic changes to this effect, but that would probably need some discussion on zulip first